product/views/PhysicalStorage.yaml

```diff
@@ -14,11 +14,12 @@ cols:
 - type
 - health_state
 - power_state
```

Comment:
This should also be removed?

Reply:
looks like it was there because I rolled back those changes.
I reintroduced them and deleted those invalid columns